forte_wrapper/allennlp/allennlp_processors.py

```diff
@@ -186,3 +186,14 @@ def _create_srl(input_pack: DataPack, tokens: List[Token],
                                         tokens[arg_span.end].end)
                 link = PredicateLink(input_pack, pred, arg)
                 link.arg_type = label
+
+    @classmethod
+    def expected_types_and_attributes(cls) -> Dict[str, Set[str]]:
+        r"""Method to add expected type for current processor input which
+        would be checked before running the processor if
+        :meth:`~forte.pipeline.Pipeline.enforce_consistency` was enabled for
+        the pipeline.
+        """
+        expectation_dict: Dict[str, Set[str]] = dict()
+        expectation_dict["ft.onto.base_ontology.Sentence"] = set()
```

Comment:
As a demonstration, probably we can make this function simpler by using the literal implementation:

```suggestion
        expectation_dict: Dict[str, Set[str]] = {
             "ft.onto.base_ontology.Sentence": set()
        }
```